content/pt-br/docs/tasks/configure-pod-container/quality-service-pod.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Configurando Qualidade do Serviço Para Pods
+content_type: task
+weight: 30
+---
+
+
+<!-- overview -->
+
+Esta página mostra como configurar os Pods para que, a eles sejam atribuídos particularmente classes de
+Qualidade de Serviço (QoS). O Kubernetes usa classes QoS para tomar decisões sobre
+agendamento e despejo de Pods.
+
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+
+<!-- steps -->
+
+## Classes QoS
+
+Quando o Kubernetes cria um Pod, ele atribui uma dessas classes de QoS ao Pod:
+
+* Guaranteed
+* Burstable
+* BestEffort
+
+## Crie um namespace
+
+Crie um namespace, assim os seus recursos criados neste exercício estarão 
+isolados do resto do seu cluster.
+
+```shell
+kubectl create namespace qos-example
+```
+
+## Crie um Pod ao qual seja atribuída uma classe de QoS `Guaranteed`
+
+Para que um Pod receba uma classe de QoS `Guaranteed`:
+
+* Todo contêiner no Pod deve ter um limite de memória e um requisito de memória.
+* Para cada contêiner no Pod, o limite de memória deve ser igual ao requisito de memória.
+* Todo contêiner no Pod deve ter um limite de CPU e um requisito de CPU.
+* Para cada contêiner no Pod, o limite da CPU deve ser igual ao requisito da CPU.
+
+Essas restrições se aplicam a contêineres de inicialização e contêineres de aplicativos, igualmente.
```

Comment:
Sugestion, we could rephrase to:
"Essas restrições se aplicam igualmente a contẽineres de inicialização bem como de aplicativos."